src/Compilers/Core/Portable/Compilation/Extensions.cs

```diff
@@ -168,7 +168,8 @@ public static DataFlowAnalysis AnalyzeDataFlow(this SemanticModel semanticModel,
         }
 
         /// <summary>
-        /// Analyze data-flow within a part of a method body. 
+        /// Analyze data-flow within a part of a method body.
+        /// note for CSharp: we handle a ConstructorInitializerSyntax like it would be a regular statement
```

Comment:
>note for CSharp: we handle a ConstructorInitializerSyntax like it would be a regular statement [](http://example.com/codeflow?start=12&length=94)

"For C# ConstructorInitializerSyntax is treated as a regular statement"?